getting-started-expressjs/index.js

```diff
@@ -0,0 +1,39 @@
+import { Metaplex, keypairIdentity, mockStorage } from "@metaplex-foundation/js-next";
+import { Connection, clusterApiUrl, PublicKey, Keypair } from "@solana/web3.js";
+import express from 'express';
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
```

Comment:
Instead of writing this as an ES Module, could we have it as a Common JS module?

That means, replacing all `import x from 'y'` by `const x = require('y')` and renaming the file to `index.cjs` to explicitly mark it as a Common JS module.

Express.js is a Node.js app and IIRC, most people use CJS and not ESM in their Node backends.

Using CJS might even remove the need for babel but I'm not sure on that one.